plugins/inputs/conntrack/README.md

```diff
@@ -36,21 +36,53 @@ For more information on conntrack-tools, see the
    ## Directories to search within for the conntrack files above.
    ## Missing directories will be ignored.
    dirs = ["/proc/sys/net/ipv4/netfilter","/proc/sys/net/netfilter"]
+   ## If true, collect conntrack stats
+   collect_stats = false
```

Comment:
```suggestion
   extra_stats = ["conntrack", "percpu"]
```

Usually we avoid adding boolean flags to a config because it makes it hard to change for future additions while keeping backwards compatibility. I assume there might be other stats we can gather, so maybe instead it should be a list of extra stats the user can enable? thoughts?